misk-launchdarkly-core/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -75,6 +78,22 @@ class LaunchDarklyFeatureFlags @Inject constructor(
     return java.lang.Enum.valueOf(clazz, result.value.toUpperCase())
   }
 
+  override fun <T> getJson(
+    feature: Feature,
+    key: String,
+    clazz: Class<T>,
+    attributes: Attributes
+  ): T {
+    checkInitialized()
+    val result = ldClient.jsonValueVariationDetail(
+        feature.name,
+        buildUser(feature, key, attributes),
+        LDValue.ofNull())
+    checkDefaultNotUsed(feature, result)
```

Comment:
What's `checkDefaultNotUsed` doing?

Reply:
basically working around the LD convention of returning a default value when the flag doesn't exist. so instead of returning the default value, we blow up instead :(